core/src/lib/util.cc

```diff
@@ -1137,3 +1146,19 @@ std::string getenv_std_string(std::string env_var)
   const char* v = (std::getenv(env_var.c_str()));
   return v ? std::string(v) : std::string();
 }
+
+
+bool pm_append(void* pm_string, const char* fmt, ...)
```

Comment:
I know, this is part of old legace code. But this ugly void* casts should be eliminated in the future. Otherwise it is always as difficult to read.

Reply:
Well, as the code is right now, this function must be defined in the same parameter as PrintMessage(void* sock, const char* fmt, ...), ...